packages/wonder-blocks-button/components/button.js

```diff
@@ -70,17 +70,18 @@ export type SharedProps = {|
     href?: string,
 
     /**
-     * Whether to use client-side navigation.
+     * Whether to avoid using client-side navigation.
      *
      * If the URL passed to href is local to the client-side, e.g.
-     * /math/algebra/eval-exprs, then it uses react-router-dom's Link
-     * component which handles the client-side navigation.
+     * /math/algebra/eval-exprs, then it tries to use react-router-dom's Link
+     * component which handles the client-side navigation. You can set
+     * `directNav` to true avoid using client-side nav entirely.
      *
      * NOTE: All URLs containing a protocol are considered external, e.g.
      * https://khanacademy.org/math/algebra/eval-exprs will trigger a full
      * page reload.
      */
-    clientNav?: boolean,
+    directNav?: boolean,
```

Comment:
I'm glad we're making client-side navigation the default.  I wonder if we should call it something less benign, e.g. `skipClientNav`.  When we were trying to brainstorm names before I was trying to think of something short but now that I think of it being a bit longer is probably feature in this case.  Also, `skipClientNav` clearly communicates what it's doing whereas I'm sure many people will have to look up what `directNav` means. 🛑

Reply:
Good suggestion - I've renamed it to `skipClientNav`